oracle-lib/README.md

```diff
@@ -25,8 +27,11 @@ project.
 
   In each case, be sure you get the version matching the versions specified
   in the `project.clj`.
-2. Put the Jars in `oracle-lib/support`
-3. Finally, perform the actual installation: `cmr install oracle-libs`.
+2. `cd oracle-lib`
+3. `mkdir support`
+4. Put the Jars in `oracle-lib/support`
+5. From `oracle-lib`, run `./support/install_oracle_jars.sh` which will
```

Comment:
Where is this script?

Reply:
The oracle README was not current with the project. Updated.